File: module_project/CalculateApp/CalculateApp/ViewController.m
//
//  ViewController.m
//  CalculateApp
//
//  Created by 小口 哲史 on 2017/03/05.
//  Copyright © 2017年 小口 哲史. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    int all1 = 10;
    int result1 = all1 / 3;
    NSLog(@"計算結果:%d",result1);
    
    int all2 = 3;
    int result2 = all2 / 5;
    NSLog(@"計算結果:%d",result2);
    
    int all3 = 0;
    int result3 = all3 / 5;
    NSLog(@"計算結果:%d",result3);
    
    ma = [NSMutableArray array];
    int count = (int)[ma count];
    NSLog(@"データ数:%d",count);
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}


@end

File: module_project/HTTPRequestApp/Request/ViewController.swift
//
//  ViewController.swift
//  Request
//
//  Created by 小口 哲史 on 2017/12/21.
//  Copyright © 2017年 小口 哲史. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
    
    @IBAction func upLoadRequest(_ sender: UIButton) {
        print("clicked")
        let getData = GetDataTask()
        getData.requestWithClosures(preTask: {
            print("preTask")
            //プログレスダイアログ表示
        }) { (dataInfo, error) in
            print("dataInfo : \(String(describing: dataInfo))")
            print("error : \(String(describing: error))")
            //プログレスダイアログ非表示
            //レスポンスデータの解析
            //エラーハンドリング
            if (error != nil) {
            }
        }
    }
}

